dali/server/daserver.cpp

```diff
@@ -377,9 +377,8 @@ int main(int argc, const char* argv[])
 #endif
         setAllocHook(true);
 
-#ifdef _CONTAINERIZED
-        serverConfig.setown(loadConfiguration(defaultYaml, argv, "dali", "DALI", "daliconf.xml", nullptr));
-#else
+        serverConfig.setown(loadConfiguration(defaultYaml, argv, "dali", "DALI", DALICONF, nullptr));
```

Comment:
Dali was unusual in conditionally calling loadConfiguration.
Wanted to change it, because wanted to standardize on how global ptree setting was picked up.

Reply:
Split off to https://track.hpccsystems.com/browse/HPCC-24409.
It will need merging before this PR.